dspace-api/src/test/resources/org/dspace/identifier/handle-forms.properties

```diff
@@ -0,0 +1,5 @@
+# Key is the identifier to be tested; value is "true" or "false":  should this
+# identifier be supported by the provider under test?
+
+http://dx.doi.org/10.14279/depositonce-5383 = false
+123456789/1 = true
```

Comment:
Seems like a good direction to add in these Unit Tests to prove out that we are handling all identifiers appropriately.  We may want to add a note that `123456789/1 = true` ONLY succeeds in this scenario because the [Unit Test framework's `handle.prefix=123456789` in its local.cfg](https://github.com/DSpace/DSpace/blob/master/dspace-api/src/test/data/dspaceFolder/config/local.cfg#L68).

To that end, we should enhance this list with other examples that should/should not be supported.  Here's a few that come to mind:

```
# Note: This next one only succeeds because handle.prefix=123456789 in Test Framework's local.cfg
123456789.1/1 = true

# Other matches that should succeed
hdl:123456789/1 = true
hdl:123.456/1.2 = true
info:hdl/12345/4000 = true
http://hdl.handle.net/20.500.1/1.2 = true
https://hdl.handle.net/987/1 = true
http://my.dspace.edu/handle/123/456 = true
https://my.dspace.edu/handle/1.23/45.6 = true

# Other matches that should fail
http://example.com/ = false
987654321/1 = false
12.3456789/1 = false
```